src/sentry/static/sentry/app/components/asyncComponent.jsx

```diff
@@ -110,7 +113,7 @@ class AsyncComponent extends React.Component {
               },
               remainingRequests: prevState.remainingRequests - 1,
               loading: prevState.remainingRequests > 1,
-              error: !!error,
+              error: prevState.error || !!error,
```

Comment:
do we want to keep `error` at all or switch to only using `errors`? or would that complicate all the component logic?

Reply:
I think it's probably ok to have the broader/simpler `error` state here and use `errors` if we need more fine grained error details.